lib/api/1.1/northbound/nodes.js

```diff
@@ -430,12 +318,7 @@ function nodesRouterFactory (
      */
 
     router.post('/nodes/:macaddress/dhcp/whitelist', rest(function(req) {
```

Comment:
These whitelist routes are actually just dead routes if you feel so inclined to delete them.